platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -113,6 +128,21 @@ struct ipsec_sa_s {
 		};
 	} hot;
 
+	/* Anti-replay window management. */
+	struct {
+		/* AR window bucket array */
+		uint64_t bucket_arr[IPSEC_ARWIN_BUCKET_MAX];
+
+		/* AR window size  */
+		uint32_t win_size;
+
+		/* Number of buckets in AR window */
+		uint16_t nb_bucket;
+
+		/* AR window lock */
+		odp_spinlock_t lock;
+	} ar;
```

Comment:
This (or at least the spin lock and the buckets) should be placed in the hot.in structure as these are frequently modified by multiple threads. See the comment in the code.

Reply:
ACK. Will move full or part of this to hot.in.